src/randgraphs.jl

```diff
@@ -148,6 +148,48 @@ function _try_creation(n::Int, k::Vector{Int}, rng::AbstractRNG)
     return edges
 end
 
+"""Creates a [Barabási–Albert model](https://en.wikipedia.org/wiki/Barab%C3%A1si%E2%80%93Albert_model) 
+random graph with `n` nodes is grown by attaching new nodes each with `m` edges that 
+are preferentially attached to existing nodes with high degree. Undirected graphs are
+created by default; use `is_directed=true` to override.
+"""
+function barabasi_albert(n::Integer, m::Integer; is_directed=false)
```

Comment:
can you rename `m` to `k` or `z`? `m` is misleading since it is often used for the total nuber of edges in the graph